### DISCUSSION TITLE
Does this start automatically?

### DISCUSSION THREAD

**Discussion by YoshiFan501** [jekhokie/raspberry-noaa-v2#658] [Q&A]
I am looking to install this on a multi purpose raspberry pi, and I am wondering if this software when installed will run at startup, or am I able to run a file at will to start up the web server? I ask since I use the same SDR for different usages at different times on this pi.

**Comment by scm2000** [comment]
Hi. I'm also new with this code. but yes, it starts automatically...  so if you want to use the SDR for listening to other things, you need to be aware that if the receive script starts for a pass and sees the SDR in use, it will exit without doing anything...  Also unfortunately since all it does is look to see if a task: rtl_fm,  is already running, even if you have two SDR's installed, it will exit even if you are not using device 0.

The code, as it is, assumes you installed on a fresh Raspberry PI, and assumes you stick with the default "pi" user.

I am in the same boat, and expect to just order a second SDR for my personal use and leave device 0 for raspberry-noaa-v2.   and so I'm going to look in to updating the code to allow that.
